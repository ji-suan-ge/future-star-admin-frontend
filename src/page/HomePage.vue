<template>
  <el-container>
    <el-header height="45px">
      <el-row justify="center" align="middle" class="hint-info-row">
          <el-col :span="23">
            <el-image :src="logoUrl" fit="fit"></el-image>
          </el-col>
          <el-col :span="1">
            <el-dropdown size="mini" placement="bottom-end" @command="handleCommand">
              <el-col>
                <el-avatar :size="30" :src="avatarUrl"></el-avatar>
                <li class="el-icon-arrow-down"></li>
              </el-col>
              <el-dropdown-menu slot="dropdown">
                <el-dropdown-item>
                  <el-row class="avatar-pop-menu-item">
                    <h3>{{ name }}</h3>
                  </el-row>
                  <el-row class="avatar-pop-menu-item">
                    @{{ account }}
                  </el-row>
                </el-dropdown-item>
                <el-dropdown-item command="b" divided>
                  <el-row class="avatar-pop-menu-item">
                    注销登录
                  </el-row>
                </el-dropdown-item>
              </el-dropdown-menu>
            </el-dropdown>
          </el-col>
      </el-row>
    </el-header>
    <el-divider></el-divider>
    <el-container>
      <el-aside width="202px">
        <el-menu :background-color="backgroundColor" :active-text-color="activeTextColor"
                 :text-color="textColor" align="middle" :default-active="defaultActive"
                 @select="changeDefaultActive" router>
              <el-menu-item index="1" :disabled="user.privilege.enrollment!==1"
                route="/home/enrollment">
                <li class="el-icon-coordinate"></li>
                招生管理
              </el-menu-item>
              <el-menu-item index="2" :disabled="user.privilege.semester!==1"
                route="/home/course">
                <li class="el-icon-document"></li>
                班级管理
              </el-menu-item>
              <el-menu-item index="3" :disabled="user.privilege.activity!==1"
                route="/home/activity">
                <li class="el-icon-sunny"></li>
                活动管理
              </el-menu-item>
              <el-menu-item index="4" :disabled="user.privilege.student!==1"
                route="/home/alumni">
                <li class="el-icon-user"></li>
                校友管理
              </el-menu-item>
              <el-menu-item index="5" :disabled="user.privilege.super!==1"
                route="/home/authority">
                <li class="el-icon-key"></li>
                权限管理
              </el-menu-item>
        </el-menu>
      </el-aside>
      <el-main>
        <router-view></router-view>
      </el-main>
    </el-container>
  </el-container>
</template>

<script>
export default {
  name: 'HomePage',
  data () {
    return {
      user: {
        privilege: {
          enrollment: '',
          semester: '',
          activity: '',
          student: '',
          super: ''
        }
      },
      textColor: '#707070',
      activeTextColor: '#409eff',
      backgroundColor: '#f9fafc',
      defaultActive: '0',
      avatarUrl: require('../assets/avatar.jpg'),
      logoUrl: require('../assets/logo.png'),
      name: '',
      identityName: '',
      account: ''
    }
  },
  created () {
    let that = this
    if (JSON.parse(sessionStorage.getItem('user') !== null)) {
      let user = JSON.parse(sessionStorage.getItem('user'))
      this.user = user
      that.name = user.name
      that.account = user.account
      if (user.privilege.super === 1) {
        that.identityName = '超级管理员'
      } else {
        that.identityName = '管理员'
      }
    }
    that.defaultActive = that.$store.getters.getActiveIndexOfNavigation
  },
  methods: {
    handleCommand (command) {
      if (command === 'b') {
        this.signOut()
      }
    },
    changeDefaultActive: async function (index) {
      let that = this
      await that.$store.dispatch('initializeStateWhileSwitch')
      await that.$store.dispatch('changeActiveIndexOfNavigation', index)
    },
    signOut: function () {
      let that = this
      that.axios.post('/administrator/logout'
      ).then(async function (response) {
        if (response.data.code === '2000') {
          sessionStorage.setItem('user', '')
          await that.$router.push('/login')
        } else {
          that.$message({
            type: 'error',
            message: '网络繁忙，请稍后重试',
            duration: 2000
          })
        }
      }).catch(function (error) {
        console.log(error)
        that.$message({
          type: 'error',
          message: '网络繁忙，请稍后重试',
          duration: 2000
        })
      })
    }
  }
}
</script>

<style scoped>
  .el-header {
    background-color: #f9fafc;
  }

  .el-menu-item {
    width: 180px;
    padding: 0;
    margin: 0;
    font-weight: 600;
    font-size: .875rem;
    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Noto Sans", Ubuntu, Cantarell, "Helvetica Neue", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
    text-align: left;
  }

  .el-menu {
    width: 200px;
    height: 100%;
  }

  .el-aside {
    height: 680px;
    background-color: #f9fafc;
  }

  a {
    text-decoration: none;
  }

  .hint-info-row {
    padding-top: 5px;
  }

  .avatar-pop-menu-item {
    width: 160px;
    height: 40px;
    padding: 10px 12px;
    font-size: .875rem;
    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Noto Sans", Ubuntu, Cantarell, "Helvetica Neue", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
    text-align: left;
  }

  .el-divider {
    margin: 0;
  }

  .sub-menu-item {
    display: inline-block;
    width: 29px;
  }

  .el-image {
    width: 100px;
    height: 30px;
  }
</style>
