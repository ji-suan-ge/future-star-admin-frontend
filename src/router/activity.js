import CreateActivityPage from '../page/CreateActivityPage'
import ShowActivityParticipantsPage from '../page/ShowActivityParticipantsPage'
import EditExistingActivityPage from '../page/EditExistingActivityPage'
import ShowActivityInformationPage from '../page/ShowActivityInformationPage'

export default [
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  },
  {
    path: '/show-participants',
    name: 'ShowActivityParticipantsPage',
    component: ShowActivityParticipantsPage
  },
  {
    path: '/edit-activity',
    name: 'EditExistingActivityPage',
    component: EditExistingActivityPage
  },
  {
    path: '/show-activity-details',
    name: 'ShowActivityInformationPage',
    component: ShowActivityInformationPage
  }
]
