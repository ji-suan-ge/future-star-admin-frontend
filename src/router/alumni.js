import ShowAlumniInformationPage from '../page/alumni-management/ShowAlumniInformationPage'
import EditAlumniInformationPage from '../page/alumni-management/EditAlumniInformationPage'
import ShowAlumniEditHistoryPage from '../page/alumni-management/ShowAlumniEditHistoryPage'
import ShowAlumniHistoryInformationPage from '../page/alumni-management/ShowAlumniHistoryInformationPage'

export default [
  {
    path: '/show-all-information',
    name: 'ShowAlumniInformationPage',
    component: ShowAlumniInformationPage
  },
  {
    path: '/edit-alumni-information',
    name: 'EditAlumniInformationPage',
    component: EditAlumniInformationPage
  },
  {
    path: '/show-edit-history',
    name: 'ShowAlumniEditHistoryPage',
    component: ShowAlumniEditHistoryPage
  },
  {
    path: '/show-history-information',
    name: 'ShowAlumniHistoryInformationPage',
    component: ShowAlumniHistoryInformationPage
  }
]
