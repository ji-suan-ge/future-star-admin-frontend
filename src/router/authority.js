import AddAdministratorPage from '../page/authority-management/AddAdministratorPage'

export default [
  {
    path: '/add-administrator',
    name: 'AddAdministratorPage',
    component: AddAdministratorPage
  }
]
