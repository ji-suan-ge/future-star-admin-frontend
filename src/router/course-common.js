import DetailOfSemesterPage from '../page/course-common/DetailOfSemesterPage'
import DetailOfClassPage from '../page/course-common/DetailOfClassPage'
import DetailOfCoursePage from '../page/course-common/DetailOfCoursePage'
import DetailOfTeacherPage from '../page/course-common/DetailOfTeacherPage'
import DetailOfSubCoursePage from '../page/course-common/DetailOfSubCoursePage'
import EditSemesterInfoPage from '../page/course-common/EditSemesterInfoPage'
import EditClassInfoPage from '../page/course-common/EditClassInfoPage'
import EditCourseInfoPage from '../page/course-common/EditCourseInfoPage'
import EditTeacherInfoPage from '../page/course-common/EditTeacherInfoPage'
import AddCoursePage from '../page/course-common/AddCoursePage'
import AddCourseResourcePage from '../page/course-common/AddCourseResourcePage'
import StudentInClass from '../page/course-common/StudentInClass'

export default [
  {
    path: '/semester-detail',
    name: 'DetailOfSemesterPage',
    component: DetailOfSemesterPage
  },
  {
    path: '/class-detail',
    name: 'DetailOfClassPage',
    component: DetailOfClassPage
  },
  {
    path: '/course-detail',
    name: 'DetailOfCoursePage',
    component: DetailOfCoursePage
  },
  {
    path: '/teacher-detail',
    name: 'DetailOfTeacherPage',
    component: DetailOfTeacherPage
  },
  {
    path: '/sub-course-detail',
    name: 'DetailOfSubCoursePage',
    component: DetailOfSubCoursePage
  },
  {
    path: '/edit-semester-info',
    name: 'EditSemesterInfoPage',
    component: EditSemesterInfoPage
  },
  {
    path: '/edit-class-info',
    name: 'EditClassInfoPage',
    component: EditClassInfoPage
  },
  {
    path: '/edit-course-info',
    name: 'EditCourseInfoPage',
    component: EditCourseInfoPage
  },
  {
    path: '/edit-teacher-info',
    name: 'EditTeacherInfoPage',
    component: EditTeacherInfoPage
  },
  {
    path: '/add-course',
    name: 'AddCoursePage',
    component: AddCoursePage
  },
  {
    path: '/add-course-resource',
    name: 'AddCourseResourcePage',
    component: AddCourseResourcePage
  },
  {
    path: '/student-in-class',
    name: 'StudentInClass',
    component: StudentInClass
  }
]
