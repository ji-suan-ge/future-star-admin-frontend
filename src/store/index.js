import Vue from 'vue'
import Vuex from 'vuex'

Vue.use(Vuex)

export default new Vuex.Store({
  state: {
    activeIndexOfNavigation: '0',
    activeNameOfCourse: 'first',
    activeNameOfEnrollment: 'first',
    activeNameOfActivity: 'first',
    activeSemesterOfCourse: '',
    semesters: '',
    currentSemester: '',
    classes: '',
    currentClass: '',
    courses: '',
    currentCourse: '',
    infoOfAnAlumni: null
  },
  mutations: {
    changeActiveIndexOfNavigation (state, activeIndex) {
      state.activeIndexOfNavigation = activeIndex
    },
    changeActiveNameOfEnrollment (state, activeName) {
      state.activeNameOfEnrollment = activeName
    },
    changeActiveNameOfCourse (state, activeName) {
      state.activeNameOfCourse = activeName
    },
    changeActiveNameOfActivity (state, activeName) {
      state.activeNameOfActivity = activeName
    },
    changeActiveSemesterOfCourse (state, activeID) {
      state.activeSemesterOfCourse = activeID
    },
    changeSemesters (state, semesters) {
      state.semesters = semesters
    },
    changeInfoOfAnAlumni (state, alumniInfo) {
      state.infoOfAnAlumni = alumniInfo
    },
    changeCurrentSemester (state, semester) {
      state.currentSemester = semester
    },
    changeClasses (state, classes) {
      state.classes = classes
    },
    changeCurrentClass (state, currentClass) {
      state.currentClass = currentClass
    },
    changeCourses (state, courses) {
      state.courses = courses
    },
    changeCurrentCourse (state, currentCourse) {
      state.currentCourse = currentCourse
    },
    initializeStateWhileLogin (state) {
      state.activeIndexOfNavigation = '0'
      state.activeNameOfCourse = 'first'
      state.activeNameOfEnrollment = 'first'
      state.activeNameOfActivity = 'first'
      state.activeSemesterOfCourse = ''
      state.semesters = ''
      state.currentSemester = ''
      state.classes = ''
      state.currentClass = ''
      state.courses = ''
      state.currentCourse = ''
      state.infoOfAnAlumni = null
    },
    initializeStateWhileSwitch (state) {
      state.activeNameOfCourse = 'first'
      state.activeNameOfEnrollment = 'first'
      state.activeNameOfActivity = 'first'
      state.activeSemesterOfCourse = ''
      state.semesters = ''
      state.currentSemester = ''
      state.classes = ''
      state.currentClass = ''
      state.courses = ''
      state.currentCourse = ''
      state.infoOfAnAlumni = null
    }
  },
  actions: {
    changeActiveIndexOfNavigation (context, activeIndex) {
      context.commit('changeActiveIndexOfNavigation', activeIndex)
    },
    changeActiveNameOfEnrollment (context, activeName) {
      context.commit('changeActiveNameOfEnrollment', activeName)
    },
    changeActiveNameOfCourse (context, activeName) {
      context.commit('changeActiveNameOfCourse', activeName)
    },
    changeActiveNameOfActivity (context, activeName) {
      context.commit('changeActiveNameOfActivity', activeName)
    },
    changeActiveSemesterOfCourse (context, activeID) {
      context.commit('changeActiveSemesterOfCourse', activeID)
    },
    changeSemesters (context, semesters) {
      context.commit('changeSemesters', semesters)
    },
    changeInfoOfAnAlumni (context, alumniInfo) {
      context.commit('changeInfoOfAnAlumni', alumniInfo)
    },
    changeCurrentSemester (context, semester) {
      context.commit('changeCurrentSemester', semester)
    },
    changeClasses (context, classes) {
      context.commit('changeClasses', classes)
    },
    changeCurrentClass (context, currentClass) {
      context.commit('changeCurrentClass', currentClass)
    },
    changeCourses (context, courses) {
      context.commit('changeCourses', courses)
    },
    changeCurrentCourse (context, currentCourse) {
      context.commit('changeCurrentCourse', currentCourse)
    },
    initializeStateWhileLogin (context) {
      context.commit('initializeStateWhileLogin')
    },
    initializeStateWhileSwitch (context) {
      context.commit('initializeStateWhileSwitch')
    }
  },
  getters: {
    getActiveIndexOfNavigation (state) {
      return state.activeIndexOfNavigation
    },
    getActiveNameOfEnrollment (state) {
      return state.activeNameOfEnrollment
    },
    getActiveNameOfCourse (state) {
      return state.activeNameOfCourse
    },
    getActiveNameOfActivity (state) {
      return state.activeNameOfActivity
    },
    getActiveSemesterOfCourse (state) {
      return state.activeSemesterOfCourse
    },
    getSemesters (state) {
      return state.semesters
    },
    getInfoOfAnAlumni (state) {
      return state.infoOfAnAlumni
    },
    getCurrentSemester (state) {
      return state.currentSemester
    },
    getClasses (state) {
      return state.classes
    },
    getCurrentClass (state) {
      return state.currentClass
    },
    getCourses (state) {
      return state.courses
    },
    getCurrentCourse (state) {
      return state.currentCourse
    }
  },
  modules: {
  }
})
