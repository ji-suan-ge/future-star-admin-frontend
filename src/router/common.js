import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import homeChildren from './home-children'

export default [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      ...homeChildren
    ]
  }
]
