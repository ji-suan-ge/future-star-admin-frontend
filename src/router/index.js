import Vue from 'vue'
import VueRouter from 'vue-router'
import LoginPage from '../page/LoginPage'
import HomePage from '../page/HomePage'
import EnrollmentManagementPage from '../page/EnrollmentManagementPage'
import CourseManagementPage from '../page/CourseManagementPage'
import ActivityManagementPage from '../page/ActivityManagementPage'
import AlumniManagementPage from '../page/AlumniManagementPage'
import AuthorityManagementPage from '../page/AuthorityManagementPage'
import CreateActivityPage from '../page/CreateActivityPage'
import AddAdministrator from '../page/AddAdministrator'
import WelcomePage from '../page/WelcomPage'

Vue.use(VueRouter)

const routes = [
  {
    path: '/',
    name: 'Root',
    redirect: '/login'
  },
  {
    path: '/login',
    name: 'LoginPage',
    component: LoginPage
  },
  {
    path: '/home',
    name: 'HomePage',
    component: HomePage,
    redirect: '/home/welcome',
    children: [
      {
        path: 'welcome',
        name: 'WelcomePage',
        component: WelcomePage
      },
      {
        path: 'enrollment-management',
        name: 'EnrollmentManagementPage',
        component: EnrollmentManagementPage
      },
      {
        path: 'course-management',
        name: 'CourseManagementPage',
        component: CourseManagementPage
      },
      {
        path: 'activity-management',
        name: 'ActivityManagementPage',
        component: ActivityManagementPage
      },
      {
        path: 'alumni-management',
        name: 'AlumniManagementPage',
        component: AlumniManagementPage
      },
      {
        path: 'authority-management',
        name: 'AuthorityManagementPage',
        component: AuthorityManagementPage
      },
      {
        path: 'add-administrator',
        name: 'AddAdministrator',
        component: AddAdministrator
      }
    ]
  },
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  }
]

const router = new VueRouter({
  routes
})

export default router
