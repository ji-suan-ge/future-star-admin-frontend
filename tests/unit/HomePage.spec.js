import { createLocalVue, mount } from '@vue/test-utils'
import HomePage from '../../src/page/HomePage'
import Vuex from 'vuex'
import ElementUI from 'element-ui'
import VueRouter from 'vue-router'
import store from '../../src/store'
import router from '../../src/router'

describe('HomePage', () => {
  const localVue = createLocalVue()
  localVue.use(Vuex)
  localVue.use(ElementUI)
  localVue.use(VueRouter)
  const wrapper = mount(HomePage, {
    store,
    router,
    localVue
  })
  test('是一个 Vue 实例', async () => {
    await expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
