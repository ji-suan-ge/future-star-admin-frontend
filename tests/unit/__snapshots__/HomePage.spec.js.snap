// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage 快照测试 1`] = `
<section class="el-container is-vertical">
  <header class="el-header" style="height: 60px;">
    <div class="hint-info-row el-row is-justify-center is-align-middle">
      <div class="el-col el-col-1"><img src="../assets/EdStarsLogo.jpg" height="50px"></div>
      <div class="el-col el-col-2 el-col-push-22">
        <div class="el-dropdown">
          <div class="el-col el-col-24 el-dropdown-selfdefine" aria-haspopup="list" aria-controls="dropdown-menu-2347" role="button" tabindex="0"><span class="el-avatar el-avatar--circle" style="height: 50px; width: 50px; line-height: 50px;"></span>
            <li class="el-icon-arrow-down"></li>
          </div>
          <ul class="el-dropdown-menu el-popper el-dropdown-menu--mini" style="display: none;" id="dropdown-menu-2347">
            <li tabindex="-1" class="el-dropdown-menu__item">
              <!---->
              <div class="avatar-pop-menu-item el-row">
                <h3></h3>
              </div>
              <div class="avatar-pop-menu-item el-row">
                @
              </div>
            </li>
            <li tabindex="-1" class="el-dropdown-menu__item el-dropdown-menu__item--divided">
              <!---->
              <div class="avatar-pop-menu-item el-row">
                修改密码
              </div>
            </li>
            <li tabindex="-1" class="el-dropdown-menu__item el-dropdown-menu__item--divided">
              <!---->
              <div class="avatar-pop-menu-item el-row">
                注销登录
              </div>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </header>
  <section class="el-container">
    <aside class="el-aside" style="width: 202px;">
      <ul role="menubar" class="el-menu" style="background-color: rgb(249, 250, 252);" align="middle"><a href="#/home/enrollment" class="">
          <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
          <li class="el-icon-coordinate"></li>
          招生管理
          </li>
        </a> <a href="#/home/course" class="">
          <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
          <li class="el-icon-document"></li>
          课程管理
          </li>
        </a> <a href="#/home/activity" class="">
          <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
          <li class="el-icon-sunny"></li>
          活动管理
          </li>
        </a> <a href="#/home/alumni" class="">
          <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
          <li class="el-icon-user"></li>
          校友管理
          </li>
        </a> <a href="#/home/authority" class="">
          <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
          <li class="el-icon-key"></li>
          权限管理
          </li>
        </a></ul>
    </aside>
    <main class="el-main">
      <section class="el-container is-vertical">
        <header class="el-header" style="height: 60px;">
          <div class="hint-info-row el-row is-justify-center is-align-middle">
            <div class="el-col el-col-1"><img src="../assets/EdStarsLogo.jpg" height="50px"></div>
            <div class="el-col el-col-2 el-col-push-22">
              <div class="el-dropdown">
                <div class="el-col el-col-24 el-dropdown-selfdefine" aria-haspopup="list" aria-controls="dropdown-menu-3174" role="button" tabindex="0"><span class="el-avatar el-avatar--circle" style="height: 50px; width: 50px; line-height: 50px;"></span>
                  <li class="el-icon-arrow-down"></li>
                </div>
                <ul class="el-dropdown-menu el-popper el-dropdown-menu--mini" style="display: none;" id="dropdown-menu-3174">
                  <li tabindex="-1" class="el-dropdown-menu__item">
                    <!---->
                    <div class="avatar-pop-menu-item el-row">
                      <h3></h3>
                    </div>
                    <div class="avatar-pop-menu-item el-row">
                      @
                    </div>
                  </li>
                  <li tabindex="-1" class="el-dropdown-menu__item el-dropdown-menu__item--divided">
                    <!---->
                    <div class="avatar-pop-menu-item el-row">
                      修改密码
                    </div>
                  </li>
                  <li tabindex="-1" class="el-dropdown-menu__item el-dropdown-menu__item--divided">
                    <!---->
                    <div class="avatar-pop-menu-item el-row">
                      注销登录
                    </div>
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </header>
        <section class="el-container">
          <aside class="el-aside" style="width: 202px;">
            <ul role="menubar" class="el-menu" style="background-color: rgb(249, 250, 252);" align="middle"><a href="#/home/enrollment" class="">
                <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
                <li class="el-icon-coordinate"></li>
                招生管理
                </li>
              </a> <a href="#/home/course" class="">
                <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
                <li class="el-icon-document"></li>
                课程管理
                </li>
              </a> <a href="#/home/activity" class="">
                <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
                <li class="el-icon-sunny"></li>
                活动管理
                </li>
              </a> <a href="#/home/alumni" class="">
                <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
                <li class="el-icon-user"></li>
                校友管理
                </li>
              </a> <a href="#/home/authority" class="">
                <li role="menuitem" tabindex="-1" class="el-menu-item" style="padding-left: 20px; color: rgb(112, 112, 112); background-color: rgb(249, 250, 252);">
                <li class="el-icon-key"></li>
                权限管理
                </li>
              </a></ul>
          </aside>
          <main class="el-main">
            <div class="el-carousel el-carousel--horizontal">
              <div class="el-carousel__container" style="height: 500px;"><button type="button" class="el-carousel__arrow el-carousel__arrow--left" style="display: none;"><i class="el-icon-arrow-left"></i></button><button type="button" class="el-carousel__arrow el-carousel__arrow--right" style="display: none;"><i class="el-icon-arrow-right"></i></button>
                <div class="el-carousel__item is-active is-animating" style="transform: translateX(0px) scale(1); webkit-transform: translateX(0px) scale(1);">
                  <!---->
                  <div class="el-image">
                    <div class="el-image__placeholder"></div>
                    <!---->
                  </div>
                </div>
                <div class="el-carousel__item" style="transform: translateX(0px) scale(1); webkit-transform: translateX(0px) scale(1);">
                  <!---->
                  <div class="el-image">
                    <div class="el-image__placeholder"></div>
                    <!---->
                  </div>
                </div>
                <div class="el-carousel__item" style="transform: translateX(0px) scale(1); webkit-transform: translateX(0px) scale(1);">
                  <!---->
                  <div class="el-image">
                    <div class="el-image__placeholder"></div>
                    <!---->
                  </div>
                </div>
                <div class="el-carousel__item" style="transform: translateX(0px) scale(1); webkit-transform: translateX(0px) scale(1);">
                  <!---->
                  <div class="el-image">
                    <div class="el-image__placeholder"></div>
                    <!---->
                  </div>
                </div>
              </div>
              <ul class="el-carousel__indicators el-carousel__indicators--horizontal">
                <li class="el-carousel__indicator el-carousel__indicator--horizontal is-active"><button class="el-carousel__button">
                    <!----></button></li>
                <li class="el-carousel__indicator el-carousel__indicator--horizontal"><button class="el-carousel__button">
                    <!----></button></li>
                <li class="el-carousel__indicator el-carousel__indicator--horizontal"><button class="el-carousel__button">
                    <!----></button></li>
                <li class="el-carousel__indicator el-carousel__indicator--horizontal"><button class="el-carousel__button">
                    <!----></button></li>
              </ul>
            </div>
          </main>
        </section>
      </section>
    </main>
  </section>
</section>
`;
